src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -11,7 +18,406 @@ internal class CodeStylePage : AbstractOptionPage
     {
         protected override AbstractOptionPageControl CreateOptionPage(IServiceProvider serviceProvider)
         {
-            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s));
+            return new GridOptionPreviewControl(serviceProvider, (o, s) => new StyleViewModel(o, s), GetCurrentEditorConfigOptionsCSharp, LanguageNames.CSharp);
+        }
+
+        internal static void Generate_Editorconfig(
+            OptionSet optionSet,
+            string language,
+            StringBuilder editorconfig)
+        {
+            GridOptionPreviewControl.Generate_Editorconfig(optionSet, language, editorconfig, GetCurrentEditorConfigOptionsCSharp);
+        }
+
+        private static void GetCurrentEditorConfigOptionsCSharp(OptionSet optionSet, StringBuilder editorconfig)
+        {
+            editorconfig.AppendLine();
+            editorconfig.AppendLine("# C# Coding Conventions");
+
+            editorconfig.AppendLine("# var preferences");
```

Comment:
These lines should likely use the localized category titles we already have for the CodeStyle UI grid.

Reply:
I tried to incorporate the ones I could find (var, parentheses), but I think quite a few are still missing. I have submitted a [bug](https://github.com/dotnet/roslyn/issues/28584) that will hopefully localize these strings.

Reply:
Note: adding localized strings is actually *really* easy :)   I'm sure someone in the room could help you out.  It's honestly only a few seconds.  So we might as well just do that as part of this PR instead of punting it.  Just my 2c here :)